grammars/lsl.cson

```diff
@@ -61,6 +61,18 @@
                 'name': 'punctuation.definition.comment.lsl'
         'match': '(//).*$\\n?'
         'name': 'comment.line.double-slash.lsl'
+    'pragma_inline':
```

Comment:
@furroy I don't understand the changes you propose here.

One the one hand you suggest `captures` groups for the match for `pragma_inline`, on the other you assign the whole match a scope `name`. Additionally your regex doesn't have groups at all, so referencing a `captures: 1: ...` group doesn't make sense.

Same with `modules`.